lib/simulation/openfisca/index.js

```diff
@@ -40,10 +40,15 @@ function calculate(situation, debug, callback) {
 }
 
 function simulate(situation, callback) {
+    var injectedRessources = _.unique(_.flatten(_.map(situation.individus, function(individu) {
+        return _.uniq(_.map(individu.ressources, function(ressource) {
+            return ressource.type;
```

Comment:
If you're simply accessing a prop, use the `String` version of the iteratee  :)
(in which case you can also improve readability by using the `collect` synonym of `map`)